File: lab4/lang-tests/test-1/typed.java
class StrongerTyped {

    public static void main(String[] args)
    {

        int tStatic = 0;
        /* Intentamos cambiar el tipo de
        la variable "tStatic"*/
        
        tStatic = "I'm a string";
        /* Esto dará un error en tiempo de compilación: 
        Se esperaba un int. */
    }
}

File: lab4/lang-tests/test-1/typed.scala
object Typed {

    def main (args: Array[String]) {
        
        var tstatic = 0;

        /* Esto dará un error en tiempo de compilación: 
        Se esperaba un int. Ademas de que no permite
        reescribir el valor */
        tstatic = "Never will be this value";
    }
}